client/question/services/SolutionHandlerServiceSpec.js

```diff
@@ -134,11 +137,14 @@ describe('SolutionHandlerService', function() {
           '    return False'
         ].join('\n');
 
+        spyOn(EventHandlerService, 'createCodeSubmitEvent');
```

Comment:
I don't think you need to add this to every single case -- can't we just have one Karma test dedicated to checking that events are handled correctly?

Reply:
Got it.